### ISSUE TITLE
¿Cúal es la licencia de este código?

### ISSUE THREAD

**Issue by josx** [prometheus-ar/vot.ar#2]
Es importante saber cual es la licencia de este código y quienes son los que tienen los derechos patrimoniales.

**Comment by prometheus-ar** [comment]
La propiedad intelectual y la patente (provisoria) son de MSA.

**Comment by josx** [comment]
Gracias por contestar.
Las patentes de software no existen en Argentina, supongo que lo que te referis es al hardware y/o a todo junto.

**Comment by prometheus-ar** [comment]
Exacto. Más específicamente me refiero a esto: https://twitter.com/beabusaniche/status/613004881016832000

**Comment by csrocha** [comment]
Dice que es softwarelibre... no me quedó claro.

https://www.youtube.com/watch?v=hwVUzvZg4GQ
